### PR TITLE
fix(utils): Removed Circular dependency in @chakra-ui/utils

### PR DESCRIPTION
**Pull request by MasterGordon**
## 📝 Description

If you bundle Chakra-UI yourself with rollup you get a warning about a Circular dependency.

## ⛳️ Current behavior (updates)

The Change fixed this warning, when bundling with rollup:
Circular dependency: ../../common/temp/node_modules/.pnpm/@chakra-ui+utils@1.8.0/node_modules/@chakra-ui/utils/dist/esm/index.js -> ../../common/temp/node_modules/.pnpm/@chakra-ui+utils@1.8.0/node_modules/@chakra-ui/utils/dist/esm/walk-object.js -> ../../common/temp/node_modules/.pnpm/@chakra-ui+utils@1.8.0/node_modules/@chakra-ui/utils/dist/esm/index.js

## 🚀 New behavior

No more Rollup warning

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by with-heart — APPROVED** [review]

**Comment by MasterGordon** [comment]
@with-heart For me it looks like Vercel failed because of some kide of timeout or other non related reasons. Is there a way to rerun the check without a dummy commit?

File: .changeset/polite-gifts-join.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/utils": minor
```
Comment (with-heart) [inline .changeset/polite-gifts-join.md:2]: I'm going to change this to `patch` since it's only a fix. `minor` would be for adding a new feature.

**Comment by with-heart** [comment]
@MasterGordon Yeah we've been having some memory issues with our Vercel deploys. Should be fine though.

Also I love your avatar!

**Comment by with-heart** [comment]
@allcontributors please add @MasterGordon for code

**Pull request merged**